src/test/java/org/folio/edge/patron/utils/PatronOkapiClientTest.java

```diff
@@ -213,7 +215,32 @@ public void testGetAccountOffsetIsWrong(TestContext context) throws Exception {
         resp -> {
           logger.info("mod-patron response body: " + resp.body());
           context.assertEquals(400, resp.statusCode());
-          context.assertEquals(wrongOffsetMessage, resp.bodyAsString());
+          context.assertEquals(String.format(WRONG_INTEGER_PARAM_MESSAGE, OFFSET, "-1"), resp.bodyAsString());
+          async.complete();
+        },
+        context::fail);
+    });
+  }
+
+  @Test
+  public void testGetAccountLimitIsWrong(TestContext context) throws Exception {
+    logger.info("=== Test getAccount - offset is wrong ===");
```

Comment:
wrong logging - should be limit instead of offset

Reply:
Changed